addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1329,8 +1334,22 @@ registry.WebsiteAnimate = publicWidget.Widget.extend({
                 this.$target.find('.modal:visible').scrollTop() :
                 this.$scrollingElement.scrollTop();
             const elTop = this._getElementOffsetTop($el[0]) - scrollTop;
-
-            const visible = this.windowsHeight > (elTop + elOffset) && 0 < (elTop + elHeight - elOffset);
+            let visible;
+            const footerEl = el.closest('.o_footer_slideout');
+            const wrapEl = this.$target[0];
+            if (footerEl && wrapEl.classList.contains('o_footer_effect_enable')) {
+                // Since the footer slideout is always in the viewport but not
+                // always displayed, the way to calculate if an element is
+                // visible in the footer is different.
+                const actualScroll = wrapEl.scrollTop + this.windowsHeight;
+                const totalScrollHeight = wrapEl.scrollHeight;
+                const heightFromFooter = this._getElementOffsetTop(el, footerEl);
+                visible = this.windowsHeight > (elTop + elOffset) && actualScroll >=
+                    totalScrollHeight - heightFromFooter - elHeight + elOffset;
```

Comment:
I think you dont need the first condition `this.windowsHeight > (elTop + elOffset) ` cause in this case the slideout is disable